flex-demo/script.js

```diff
@@ -0,0 +1,17 @@
+
+
+document.getElementById("justify-content").addEventListener("change", function() {
+    const value = this.value;
```

Comment:
directly assign this.value to below line

Reply:
done 👍